snippets/stableSort.md

```diff
@@ -0,0 +1,29 @@
+### stableSort
+
+Performs stable sort. Useful in Chrome and NodeJS.
+
+Use `Array.map()` to pair each element of the input array with its corresponding index. Then use `Array.sort()` and a user provided `compare()` function. If the items are equal, sort them by their initial index in the input array. Lastly use `Array.map()` to convert back to the initial array items.
+Returns new array without modifying the initial one.
+
+```js
+var stableSort = (arr, compare) =>
```

Comment:
change to const deceleration for the snippet, change name to reflect snippet name when changed